crates/common/src/contracts.rs

```diff
@@ -45,6 +45,13 @@ impl ContractsByArtifact {
                     let bytecode = artifact.bytecode.and_then(|b| b.into_bytes())?;
                     let deployed_bytecode =
                         artifact.deployed_bytecode.and_then(|b| b.into_bytes())?;
+
+                    if bytecode.is_empty() {
```

Comment:
nit: add small comment on what empty bytecode means, just like the pr description